spread.yaml

```diff
@@ -63,7 +63,7 @@ prepare: |
   else
     tests.pkgs remove lxd
   fi
-  snap install lxd --channel=5.9/stable
+  snap install lxd --channel=5.11/stable
```

Comment:
`5.11/stable` doesn't have anything in it. We can do `5.10/stable`, `latest/stable` or `5.11/candidate`, or we can do the workaround from your launchpad bug.

Reply:
Nice catch, thanks.  Trying `latest/stable` to see if that passes.